lib/listener/helpers.js

```diff
@@ -13,6 +13,10 @@ module.exports = function () {
   const runHelpersHook = (hook, param) => {
     if (store.dryRun) return;
     Object.keys(helpers).forEach((key) => {
+      if (!helpers[key][hook]) {
+        error(`${key}'s hook value is: ${helpers[key][hook]}`);
```

Comment:
```suggestion
```
I don't see any point in this message. You checked for this value and it is null or undefined
better to remove this check so any class could be a helper